modules/rest/src/test/java/org/apache/ignite/internal/rest/configuration/JsonMaskerTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.rest.configuration;
+
+import static java.util.Collections.emptySet;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+class JsonMaskerTest {
+
+    private final Set<String> keysToMask = Set.of("credentials");
+    private final JsonMasker jsonMasker = new JsonMasker();
```

Comment:
Why JsonMasker can't set `keysToMask` in constructor? Seems like it is a part of object configuration and can be put to the constructor instead of propagating it in the method signature.

Reply:
It can, but in that case, we have to create JsonMasker for each set of rules.